py3status/modules/scratchpad_counter.py

```diff
@@ -4,7 +4,11 @@
 
 Configuration parameters:
     cache_timeout: How often we refresh this module in seconds (default 5)
-    format: string to print (default '{counter} ⌫')
+    format: string to print
+        To make empty windows go away, use 'format = '[\?not_zero {counter} ⌫]'
+        To print something else instead, use 'format = '[\?not_zero {counter} ⌫]|Empty`
```

Comment:
This does not belong here.  This is about the formatter not anything to do with this module.

It does need documenting, but imho, we need to get all the modules using format correctly first.